### ISSUE TITLE
New view for RCB?

### ISSUE THREAD

**Issue by Slipass** [maloep/romcollectionbrowser#419]
Hi maloep,

I don't know if this is the right place for the threat but I wanted to give some ideas for new views.

Since I am a big fan of RCB, I would be happy if maybe one or two inspirations would be added to the RCB site.

[https://imgur.com/qNqoVAu](url)

[https://imgur.com/15MZBOa](url)

[https://imgur.com/c2FgjHH](url)

[https://imgur.com/oKTPRge](url)

[https://imgur.com/3ks6hy9](url)

[https://imgur.com/QD0pbKN](url)

[https://imgur.com/rZYvGBM](url)

[https://imgur.com/ltHDKMy](url)

**Comment by maloep** [comment]
Thanks for the input. I like the views with the xbox covers. 

But unfortunately I don't think that I can add something like this. I can't create these dvd cases per code. So this would require broad availability of this artwork type for several systems. Do you know a source where these are available? Otherwise I think it is not possible to create these views (at least not with my skinning skills).

The last two views with the 3D box covers should already be possible with RCB. There already is an artwork type 3dbox in RCB. But this also requires that you have these boxes as artwork in your collection.

**Comment by Slipass** [comment]
> Thanks for the input. I like the views with the xbox covers.
> 
> But unfortunately I don't think that I can add something like this. I can't create these dvd cases per code. So this would require broad availability of this artwork type for several systems. Do you know a source where these are available? Otherwise I think it is not possible to create these views (at least not with my skinning skills).
> 
> The last two views with the 3D box covers should already be possible with RCB. There already is an artwork type 3dbox in RCB. But this also requires that you have these boxes as artwork in your collection.

Thanks for the feedback.

I do not currently have a source for these.
The Screenshot is from my old xbox360 mod. It is named "Xbox 360 Freestyle Dash".
I found a link in the web:
https://www.youtube.com/watch?v=ckqaA3aOFdo
https://www.konsolenumbau.at/xbox360_info.php?id=3
But you wouldn't have to recreate the views exactly the same way.

Maybe just the animations are enough.

The last two screenshots are from my system.
But the screenshots are older. The views were from Skin Madnox itself. There were .xml adjustments to RCB in the Skin Order.
Since the new versions of RCB the xml files are no longer supported.
So I had to deactivate them.

The advantage with the old views was that the covers were scaled higher.
I use for all my ROMS 3D covers and with the current view of RCB the scaling is a bit small.

Here the current view (too small covers)
![](https://i.imgur.com/xyjOuNm.jpg)

What I would find really great would be a better choice of emulators.
Maybe with more graphics? 

Or maybe at least at the bottom left, where the system is called, a graphic from the console?

**Comment by Slipass** [comment]
Maybe something like this

![](https://i.imgur.com/gpRnhmr.jpg)

**Comment by maloep** [comment]
Atm RCB has no console information and artwork that is scraped. It is planned to add this in a later release and also add a start page where you can browse by console. But this will need a lot of time and there are many issues that I want to fix first before I start working on this one.

Scaling is an issue since the beginning. I am still thinking about better solutions but this is also a feature that will be adressed in a later release. I don't think that it will make it into one of the next releases.

**Comment by Kupo91** [comment]
> and also add a start page where you can browse by console

Hi malte, I hope this will be just an extra option which can be disabled because I like having all my games in one view without being seperated by consoles. This is one of the reasons why I use RCB instead of Emulationstation, Pegasus etc.

**Comment by maloep** [comment]
Yes, this is an important feature for me too. The console browsing could be the new start page with console info and statistics. When you jump into a console you will have the same page as you have now with console filter that you can still set to "All". 

Additionally it should be configurable if you want to use this start page or if you want to start with the Games page.

**Comment by Kupo91** [comment]
Here is another view from Aeon MQ, which could be added:

![Wall](https://user-images.githubusercontent.com/41426906/86189961-5581e300-bb43-11ea-83ea-1a757c74646e.jpg)

The view is called "Wall"
